Assets/MixedRealityToolkit.Extensions/Providers/Experimental/UnityAR/UnityARCameraSettings.cs

```diff
@@ -79,9 +81,10 @@ public UnityARCameraSettingsProfile SettingsProfile
             }
         }
 
-        bool isInitialized = false;
-
 #if !(WINDOWS_UWP && !ENABLE_IL2CPP)
```

Comment:
Do we still need these with C# Projects turned off?

Reply:
In theory, yes. Since Unity would compile the asmdefs using the .net backend, correct?

Reply:
It shouldn't, since the asmdef is marked as iOS and Android only

Reply:
oh yeah. i see where you are going now :)

Reply:
Might need to instead wrap it in iOS and Android checks though, since it's marked as valid in the Editor as well.
That way, devs who only consume MRTK via the Windows targets aren't required to import the ARFoundation package if they install our extensions.

Reply:
I also wonder if we should redefine the UnityARCameraSettingsProfile a bit, so we don't directly serialize ARFoundation types? And wrapping serialized fields in `#if` is what leads to https://github.com/microsoft/MixedRealityToolkit-Unity/issues/6054 and https://github.com/microsoft/MixedRealityToolkit-Unity/issues/3971

Reply:
done